src/components/Vote.vue

```diff
@@ -62,6 +62,19 @@
           <div class="button demon" @click="finish">Close</div>
         </div>
       </template>
+      <template v-if="!session.isSpectator">
+        <div
+          class="button-group"
+          v-if="session.lockedVote && !session.isVoteInProgress"
+        >
+          <div class="button demon" @click="setOnBlock">
+            Put on block
+          </div>
+          <div class="button demon" @click="emptyBlock">
+            Empty block
+          </div>
+        </div>
+      </template>
```

Comment:
This should be rolled into the "Close" button, splitting it into 3 options:
- Close
- Close & Mark
- Close & Remove Mark

(might be a challenge to make it look pretty)

Reply:
I tried this but it gets pretty wide with then 5 buttons in a line - not sure this is viable.

Reply:
Yeah, I think a separate line might be best, or maybe some other design. If I have some time this week I'll play around a bit with it as well.